oras/auth/token.py

```diff
@@ -66,6 +65,7 @@ def authenticate_request(
             return headers, False
 
         # If we have a token, set auth header (base64 encoded user/pass)
+        # Else if
```

Comment:
Did you mean to write another comment here?

Reply:
I tried to modify this function, but in the end i didn't